### DISCUSSION TITLE
ORM and AWS SDK RDSDATA

### DISCUSSION THREAD

**Discussion by per-oestergaard** [SeaQL/sea-orm#598] [Q&A]
I wrote the AWS team here: https://github.com/awslabs/aws-sdk-rust/discussions/489

I am looking for an ORM for my Rust code. I came across this repo but I cannot figure out whether I can combine SeaORM with RDSDATA as SeaORM might need a "real" TCP port connection and not the HTTP API provided by RDSDATA.

An example of how RDSDATA is used across HTTP is here: https://github.com/awslabs/aws-sdk-rust/blob/d9ce0025afe4d0490be458170e05ddab15a4e66c/examples/rdsdata/src/bin/rdsdata-helloworld.rs#L57

Any info on whether this is possible would be highly appreciated.

**Comment by billy1624** [comment]
Hey @per-oestergaard, sorry for the delay. I guess you can connect to RDSDATA with a connection string?
- https://github.com/launchbadge/sqlx/issues/1376
